lib/betagouvbot/mail.rb

```diff
@@ -5,42 +5,65 @@
 
 module BetaGouvBot
   class Mail
-    attr_accessor :subject
-    attr_accessor :recipients
+    class << self
+      def rules
```

Comment:
Why not? As of now, rules aren't anything more than a collection of ```Mail``` objects (nothing has been changed behaviour wise: it's still just a collection of decorated content).